test/dispatchers/keyDispatcherTests.ts

```diff
@@ -22,6 +22,34 @@ describe("Dispatchers", () => {
       assert.isTrue(keyDowned, "callback when a key was pressed");
 
       ked.offKeyDown(callback); // clean up
+
+      keyDowned = false;
+      $("body").simulate("keydown", { keyCode: keyCodeToSend });
+      assert.isFalse(keyDowned, "nothing happens when a key was pressed");
     });
+
+    it("triggers callback on keyup", () => {
+          var ked = Plottable.Dispatchers.Key.getDispatcher();
+
+          var keyCodeToSend = 65;
+
+          var keyUped = false;
+          var callback = (code: number, e: KeyboardEvent) => {
+            keyUped = true;
+            assert.strictEqual(code, keyCodeToSend, "correct keycode was passed");
+            assert.isNotNull(e, "key event was passed to the callback");
+          };
+
+          ked.onKeyUp(callback);
+
+          $("body").simulate("keyup", { keyCode: keyCodeToSend });
+          assert.isTrue(keyUped, "callback when a key was release");
+
+          ked.offKeyUp(callback); // clean up
```

Comment:
I know it's been used above as well and we don't have a tslint rule for this, but let's avoid comments at the end of the line (move above)